.travis.yml

```
@@ -26,5 +29,24 @@ install:
     - cd flex
     - composer config extra.symfony.allow-contrib true
 
-script:
-    - composer req --ignore-platform-reqs $(echo $PACKAGES)
+before_script:
+  - |
+    install_package() {
+        composer require --ignore-platform-reqs $@;
+        EXIT_CODE=$?;
+
+        MESSAGE="";
+        if [[ EXIT_CODE -eq 0 ]]; then
+            exit 0;
+        elif [[ EXIT_CODE -eq 1 ]]; then
+            MESSAGE="Could not install package and configure package.";
+        elif [[ EXIT_CODE -eq 2 ]]; then
+            MESSAGE="Could not resolve dependencies.\n# You can ignore this error if your package does not support Symfony $SKELETON_VERSION";
+        fi
```

```suggestion
            MESSAGE="Could not resolve dependencies.\n# You can ignore this error if your package does not support Symfony $SKELETON_VERSION";
            exit 0
        fi
```

Shouldn't we make the job green when the package could not be installed?

---

We are not. We always exit with the same exit code as composer

---

I dont think we should mark the job as green. Or should we....
I dont want to have 3 green travis job but the package could not be installed .